Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -706,4 +706,18 @@ def test_irb
 
     assert_match "Interactive Homebrew Shell", cmd("irb", irb_test)
   end
+
+  def test_switch
+    assert_match "Usage: brew switch <name> <version>", cmd_fail("switch")
+
+    setup_test_formula "testball", <<-EOS.undent
+      keg_only "just because"
+    EOS
+
+    cmd("install", "testball")
+    testball_keg = HOMEBREW_CELLAR/"testball"
+    FileUtils.cp_r testball_keg/"0.1", testball_keg/"0.2"
```

Comment:
It's a bit of a technicality, but this should be`testball_rack`. Here's why:

- `HOMEBREW_CELLAR` - cellar (home to all installed formulae)
- `HOMEBREW_CELLAR/name` – rack (home to all installed versions of a formula)
- `HOMEBREW_CELLAR/name/pkg_version` - keg (home to a specific version plus revision of a formula)

The `name` and `pkg_verison` placeholders above are references to the corresponding `Formula` instance methods.